patroni/postgresql.py

```diff
@@ -78,6 +78,7 @@ def __init__(self, config):
         self.replication_slots = []  # list of already existing replication slots
         self.retry = Retry(max_tries=-1, deadline=5, max_delay=1, retry_exceptions=PostgresConnectionException)
 
+        self._creating_replica = False
```

Comment:
Good idea, so in addition to the running and stopped we should also have something like "initializing"